src/cares_wrap.cc

```diff
@@ -460,27 +558,33 @@ void AresEnsureServers(Environment* env) {
       servers[0].tcp_port != 0 ||
       servers[0].udp_port != 0) {
     ares_free_data(servers);
-    env->set_cares_is_servers_default(false);
+    is_servers_default_ = false;
     return;
   }
 
   ares_free_data(servers);
   servers = nullptr;
 
   /* destroy channel and reset channel */
-  ares_destroy(channel);
+  ares_destroy(channel_);
 
-  SetupCaresChannel(env);
+  Setup();
 }
 
 
 class QueryWrap : public AsyncWrap {
  public:
-  QueryWrap(Environment* env, Local<Object> req_wrap_obj)
-      : AsyncWrap(env, req_wrap_obj, AsyncWrap::PROVIDER_QUERYWRAP) {
-    if (env->in_domain())
-      req_wrap_obj->Set(env->domain_string(), env->domain_array()->Get(0));
+  QueryWrap(ChannelWrap* channel, Local<Object> req_wrap_obj)
+      : AsyncWrap(channel->env(), req_wrap_obj, AsyncWrap::PROVIDER_QUERYWRAP),
+        channel_(channel) {
+    if (env()->in_domain())
+      req_wrap_obj->Set(env()->domain_string(), env()->domain_array()->Get(0));
```

Comment:
It’s not really new code, but yes, I’m changing this to the non-deprecated version of `Get` so that it “only” crashes.